Libraries/Image/RCTImageCache.m

```diff
@@ -38,10 +38,10 @@ - (instancetype)init
 {
   if (self = [super init]) {
     _decodedImageCache = [NSCache new];
-#if !TARGET_OS_OSX // TODO(macOS GH#774)
     _decodedImageCache.totalCostLimit = 20 * 1024 * 1024; // 20 MB
     _cacheStaleTimes = [NSMutableDictionary new];
```

Comment:
there are other usage of _cachestatletimes that we are not undefing with this change.  for example
```
#if !TARGET_OS_OSX // TODO(macOS GH#774)
- (void)clearCache
{
  [_decodedImageCache removeAllObjects];
  @synchronized(_cacheStaleTimes) {
    [_cacheStaleTimes removeAllObjects];
  }
}
```
how are we clearing things then?

Reply:
See comment here, this should be fine https://github.com/microsoft/react-native-macos/pull/1272#issuecomment-1209950757